egui_demo_lib/src/apps/demo/widget_gallery.rs

```diff
@@ -124,6 +128,29 @@ impl WidgetGallery {
         });
         ui.end_row();
 
+        ui.label("");
+        ui.horizontal(|ui| {
+            if ui
+                .add(egui::RadioButton::new(*radio == Enum::First, "First").label_on_left(true))
+                .clicked()
+            {
+                *radio = Enum::First
+            };
+            if ui
+                .add(egui::RadioButton::new(*radio == Enum::Second, "Second").label_on_left(true))
+                .clicked()
+            {
+                *radio = Enum::Second
+            };
+            if ui
+                .add(egui::RadioButton::new(*radio == Enum::Third, "Third").label_on_left(true))
+                .clicked()
+            {
+                *radio = Enum::Third
+            };
+        });
+        ui.end_row();
+
```

Comment:
The widget gallery is to showcase one of each widget, not do showcase all variants of each widget.

Please move this to e.g. `egui_demo_lib/src/apps/demo/misc_demo_window.rs` instead!